neurom/core/types.py

```diff
@@ -47,48 +48,107 @@ class NeuriteIter(OrderedEnum):
     NRN = 2
 
 
-class Subtypes:
+class NeuriteSubtype(int):
+    _ALL = 32
+    _BASE = 100
+    _COMPOSITE = 99
```

Comment:
Is this needed?

Reply:
Ah no you're right, `_COMPOSITE` is no more used, it was just for a test.